contracts/Peeranha.sol

```diff
@@ -49,9 +50,13 @@ contract Peeranha is IPeeranha, Initializable {
         SecurityLib.setupRole(userContext, SecurityLib.DEFAULT_ADMIN_ROLE, msg.sender);
     }
 
-    function getRatingToReward(address user, uint16 rewardPeriod) external view override returns(int32) {
-        RewardLib.PeriodRating storage userPeriod = RewardLib.getUserPeriodRating(userContext.userRewards, user, rewardPeriod);
-        return userPeriod.ratingToReward;
+    function getUserRewardCommunities(address user, uint16 rewardPeriod) external override view returns(uint32[] memory) {
+        require(
```

Comment:
Why need this check in the view function? I suggest to remove it and just return empty array.